packages/jaeger-ui/src/components/TracePage/index.js

```diff
@@ -58,6 +59,10 @@ type TracePageProps = {
   history: RouterHistory,
   id: string,
   trace: ?FetchedTrace,
+  isEmbed?: boolean,
+  minimap?: boolean,
+  details?: boolean,
+  mapCollapsed?: boolean,
```

Comment:
What is `minimap` vs `mapCollapsed`?

Reply:
mapCollapsed is triggered when the user click in the trace tittle this changes the arrow too.

Minimap remove the map, the arrow and this click.

I think that we should keep this, if I use mapCollapsed the user can click and open the minimap unless that we want to use this behavior. What do you think?